sample/kotlin/src/main/kotlin/com/datadog/android/sample/SampleApplication.kt

```diff
@@ -35,7 +35,7 @@ import timber.log.Timber
 
 class SampleApplication : Application() {
 
-    private val tracedHosts = listOf("shopist.io")
+    private val tracedHosts = listOf("shopist.io,source.unsplash.com")
```

Comment:
No we don't want to trace unsplash. Tracing should only be used for request which hosts we control.